lxd/sys/apparmor.go

```diff
@@ -80,7 +80,7 @@ func (s *OS) initAppArmor() []cluster.Warning {
 	   the label will have the suffix " (unconfined)" */
 	if profile != "unconfined" && profile != "" && !strings.HasSuffix(profile, "(unconfined)") {
 		if s.AppArmorAvailable {
-			logger.Warnf("Per-container AppArmor profiles are disabled because LXD is already protected by AppArmor via profile '%s'", profile)
+			logger.Warnf("Per-container AppArmor profiles are disabled because LXD is already protected by AppArmor via profile %q", profile)
```

Comment:
We would normally expect a change on an unmerged commit to be rebased and forced push rather than a new commit fixing the issue. This way we only land the final version of a PR rather than merging its evolution during the review process (we dont do squash merges). Please can this be rebased into the original commit that introduced the ` via profile '%s'` bit. Thanks

Reply:
Sure - done!

Reply:
Thanks!